gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/assets/css/main.css

```diff
@@ -222,7 +222,7 @@ body, body * {
     flex-grow: 0;
 }
 
-form div {
+form div:not(.iti__flag-container, .iti__selected-flag, .iti__flag, .iti__arrow) {
```

Comment:
I wonder if it's not clearer to have a specific section for the intl-tel-input components :

```
.iti__flag-container, .iti__selected-flag, .iti__flag, .iti__arrow {
  width: auto;
}
```

Reply:
simplified and works in chrome, firefox and safari with 

`.iti__flag-container {
  width: auto;
}`